ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -453,6 +480,11 @@ public void onCompletion(String result, Exception exception) {
       this.getCallback = callback;
     }
 
+    SecurityProcessRequestCallback(RestRequest restRequest, RestResponseChannel restResponseChannel) {
```

Comment:
add a javadoc to say that this is used for get requests that do not contact the router.